src/index.js

```diff
@@ -0,0 +1,34 @@
+const BigNumber = require("bignumber.js");
+
+const { getInputOutputFiles, parseCsvFile, writeJson } = require("./utils");
+const GNO_ADDRESS = "0x6810e776880C02933D47DB1b9fc05908e5386b96";
+
+function toPayment(leaderBoardItem) {
+  const { trader, gno_estimation: gnoEstimation } = leaderBoardItem;
+
+  return {
+    amount: gnoEstimation,
```

Comment:
minor suggestion: rename `gnoEstimation` to `amount` and save a few chars :grin: